docs/source/reference/builtin-devices.rst

```diff
@@ -44,18 +44,51 @@ EpicsScaler
 -----------
 
 Create an ``EpicsScaler`` to control an EPICS `scaler record
-<http://www.aps.anl.gov/bcda/synApps/std/scalerRecord.html>`_:
+<https://htmlpreview.github.io/?https://github.com/epics-modules/scaler/blob/master/documentation/scalerRecord.html>`_:
 
 .. code-block:: python
 
     from ophyd import EpicsScaler
     scaler = EpicsScaler('XF:28IDC-ES:1{Sclr:1}', name='tth')
 
+Note that :ref:`ScalerCH` is an alternative ``ophyd`` representation of
+the EPICS `scaler record.
```

Comment:
```suggestion
the EPICS' scaler record.
```

Reply:
This one is for the older commit. Please disregard.

Reply:
That's not the fix (it is not punctuation). It's part of a reference on the next lines.  I'll look later today.